resources/views/users/reserved/join_users.blade.php

```diff
@@ -7,12 +7,11 @@
 @section('content')
 <div class="container">
     <div class="form w-50 mx-auto p-5">
-    <a href="{{ route('users.reserved.show.details') }}"><i class="fa-solid fa-xmark fa-pull-right mt-3 me-4 text-secondary"></i></a>
+    <a href="javascript:void(0);" onclick="history.back();"><i class="fa-solid fa-xmark fa-pull-right mt-3 me-4 text-secondary"></i></a>
```

Comment:
you can add this in one script tag

Reply:
Am I correct you said that I need to create a new js file?

Reply:
or do you mean to code as follows ?

<a href="javascript:void(0);" onclick="goBack();"><i class="fa-solid fa-xmark fa-pull-right mt-3 me-4 text-secondary"></i></a>
        <script>
            function goBack() {
                history.back();
            }
        </script>

Reply:
@section('script')
    <script>
        function goBack() {
            history.back();
        }
    </script>
@endsection

I add this one, is it ok?